x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -286,13 +286,25 @@ export const AddExceptionModalWrapper: React.FC<AddExceptionModalWrapperProps> =
     }
   }, [data?.hits.hits, isLoadingAlertData]);
 
+  /**
+   * This should be re-visited after UEBA work is merged
+   */
+  const useRuleIndices = useMemo(() => {
+    if (enrichedAlert != null && enrichedAlert['kibana.alert.rule.parameters']?.index != null) {
+      return enrichedAlert['kibana.alert.rule.parameters'].index as unknown as string[];
+    } else if (enrichedAlert != null && enrichedAlert?.signal?.rule?.index != null) {
+      return enrichedAlert.signal.rule.index as unknown as string[];
```

Comment:
`as unknown as string[]` is required because the flattening of the type ends up casting everything as a `string`, but the `ruleIndices` prop of the `AddExceptionModal` component requires `string[]`.

Reply:
can you do:
`return isArray(enrichedAlert.signal.rule.index) ? enrichedAlert.signal.rule.index : [enrichedAlert.signal.rule.index];`